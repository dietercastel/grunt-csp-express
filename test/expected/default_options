{
    "default-src": [],
    "script-src": [],
    "object-src": [],
    "style-src": [],
    "img-src": [],
    "media-src": [],
    "frame-src": [],
    "font-src": [],
    "connect-src": []
}