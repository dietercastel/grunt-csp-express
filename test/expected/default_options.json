{
    "default-src": "'none'",
    "script-src": "'none'",
    "object-src": "'none'",
    "style-src": "'none'",
    "img-src": "'none'",
    "media-src": "'none'",
    "frame-src": "'none'",
    "font-src": "'none'",
    "connect-src": [
        "'self'"
    ]
}